src/Workspaces/Core/Portable/Storage/SQLite/Interop/NativeMethods.cs

```diff
@@ -55,7 +56,7 @@ public static SafeSqliteStatementHandle sqlite3_prepare_v2(SafeSqliteHandle db,
             }
         }
 
-        public static SafeSqliteBlobHandle sqlite3_blob_open(SafeSqliteHandle db, string sdb, string table, string col, long rowid, int flags, out Result result)
+        public static SafeSqliteBlobHandle sqlite3_blob_open(SafeSqliteHandle db, utf8z sdb, utf8z table, utf8z col, long rowid, int flags, out Result result)
```

Comment:
this avoids us having to convert to/from strings and utf8zs for these three common values that we're always passing along.  but it required a little hackery further along.